AutocompleteClient/FW/Logic/Result/Search/SearchResult.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SearchResult.swift
+//  AutocompleteClient
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+struct SearchResult {
+    let id: String
+    let value: String
+    let url: String?
+    let price: String?
+    let quantity: String?
+    let imageURL: String?
+    let facets: [SearchResultFacet]?
+    let groups: [CIOGroup]?
+    
```

Comment:
This data structure isn't quite right.  A search result will have three properties:  
```
value: string
data: object
matched_terms: string array
```

and the data property will have:
```
description: string
id: string
url: string
image_url: string
groups: object array
facets: object array
metadata: object
```